router/http/src/main/scala/io/buoyant/router/Http.scala

```diff
@@ -1,13 +1,14 @@
 package io.buoyant.router
 
+import java.net.SocketAddress
+
 import com.twitter.finagle.client.StackClient
 import com.twitter.finagle.http.{Request, Response, TlsFilter}
 import com.twitter.finagle.param.ProtocolLibrary
 import com.twitter.finagle.server.StackServer
 import com.twitter.finagle.{Http => FinagleHttp, Server => FinagleServer, http => fhttp, _}
 import io.buoyant.router.Http.param.HttpIdentifier
-import io.buoyant.router.http.{ForwardedFilter, MethodAndHostIdentifier, StripConnectionHeader}
-import java.net.SocketAddress
+import io.buoyant.router.http.{DiscardProxySideEffects, ForwardedFilter, MethodAndHostIdentifier, StripConnectionHeader}
```

Comment:
super minor: can yiou please sort imports without breaking out sections.

From [Effective Scala](https://twitter.github.io/effectivescala/)
> _Sort import lines alphabetically_
> This makes it easy to examine visually, and is simple to automate.

IntelliJ, etc have settings to configure this.